tests/annotations_test.go

```diff
@@ -0,0 +1,44 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// ensures services.annotations values are applied to both coderd deployment &
+// service
+func TestAnnotations(t *testing.T) {
+	t.Parallel()
+
+	chart := LoadChart(t)
+
+	expected := map[string]string{}
```

Comment:
We definitely should be testing with actual values (and perhaps have a separate test for empty annotations, but I don't think that's as necessary).

I would also recommend testing with three `expected` vars (i.e. `expectedCoderd`, `expectedCoderdService`, `expectedTimescale`) with completely different values so we can make sure the correct values are being used in the correct places.

Also you'll need to add a test for backwards compatibility